src/Employer/ESFA.DAS.EmployerFeedbackEmail.Data/Repositories/EmployerFeedbackRepository.cs

```diff
@@ -10,7 +10,7 @@ namespace ESFA.DAS.ProvideFeedback.Data.Repositories
 {
     public class EmployerFeedbackRepository : IEmployerFeedbackRepository
     {
-        private int _commandTimeoutSeconds = 120;
+        const int CommandTimeoutSeconds = 120;
```

Comment:
The `private` modifier can remain. Member variables are private by default, but it's good to be explicit
```suggestion
        private const int CommandTimeoutSeconds = 120;
```

Reply:
done